apps/publikator/lib/settings.js

```diff
@@ -1,39 +1,38 @@
-const ENV =
-  typeof window !== 'undefined' && window.__NEXT_DATA__
-    ? window.__NEXT_DATA__.env
-    : process.env
-
-export const PUBLIC_BASE_URL = ENV.PUBLIC_BASE_URL
-
-export const LOCALE = ENV.LOCALE
-export const GITHUB_ORG = ENV.GITHUB_ORG
-export const API_URL = ENV.API_URL
-export const API_UNCOMMITTED_CHANGES_URL = ENV.API_UNCOMMITTED_CHANGES_URL
-export const API_WS_URL = ENV.API_WS_URL
-export const API_AUTHORIZATION_HEADER = ENV.API_AUTHORIZATION_HEADER
-export const FRONTEND_BASE_URL = ENV.FRONTEND_BASE_URL
-
-export const PIWIK_URL_BASE = ENV.PIWIK_URL_BASE
-export const PIWIK_SITE_ID = ENV.PIWIK_SITE_ID
-
-export const TEMPLATES = ENV.TEMPLATES
-export const REPO_PREFIX = ENV.REPO_PREFIX
+export const PUBLIC_BASE_URL = process.env.NEXT_PUBLIC_PUBLIC_BASE_URL
+
+export const LOCALE = process.env.NEXT_PUBLIC_LOCALE
+export const GITHUB_ORG = process.env.NEXT_PUBLIC_GITHUB_ORG
+export const API_URL = process.env.NEXT_PUBLIC_API_URL
+export const API_UNCOMMITTED_CHANGES_URL =
+  process.env.NEXT_PUBLIC_API_UNCOMMITTED_CHANGES_URL
+export const API_WS_URL = process.env.NEXT_PUBLIC_API_WS_URL
+export const API_AUTHORIZATION_HEADER =
+  process.env.NEXT_PUBLIC_API_AUTHORIZATION_HEADER
+export const FRONTEND_BASE_URL = process.env.NEXT_PUBLIC_FRONTEND_BASE_URL
+
+export const PIWIK_URL_BASE = process.env.NEXT_PUBLIC_PIWIK_URL_BASE
+export const PIWIK_SITE_ID = process.env.NEXT_PUBLIC_PIWIK_SITE_ID
+
+export const TEMPLATES = process.env.NEXT_PUBLIC_TEMPLATES
+export const REPO_PREFIX = process.env.NEXT_PUBLIC_REPO_PREFIX
 export const TEMPLATE_PREFIX = 'template-'
 
-export const SG_COLORS = ENV.SG_COLORS
-export const SG_FONT_STYLES = ENV.SG_FONT_STYLES
-export const SG_FONT_FACES = ENV.SG_FONT_FACES
-export const SG_LOGO_PATH = ENV.SG_LOGO_PATH
-export const SG_LOGO_VIEWBOX = ENV.SG_LOGO_VIEWBOX
-export const SG_BRAND_MARK_PATH = ENV.SG_BRAND_MARK_PATH
-export const SG_BRAND_MARK_VIEWBOX = ENV.SG_BRAND_MARK_VIEWBOX
-export const SG_DYNAMIC_COMPONENT_BASE_URLS = ENV.SG_DYNAMIC_COMPONENT_BASE_URLS
+export const SG_COLORS = process.env.SG_COLORS
+export const SG_FONT_STYLES = process.env.SG_FONT_STYLES
+export const SG_FONT_FACES = process.env.SG_FONT_FACES
+export const SG_LOGO_PATH = process.env.SG_LOGO_PATH
+export const SG_LOGO_VIEWBOX = process.env.SG_LOGO_VIEWBOX
+export const SG_BRAND_MARK_PATH = process.env.SG_BRAND_MARK_PATH
+export const SG_BRAND_MARK_VIEWBOX = process.env.SG_BRAND_MARK_VIEWBOX
+export const SG_DYNAMIC_COMPONENT_BASE_URLS =
+  process.env.SG_DYNAMIC_COMPONENT_BASE_URLS
```

Comment:
I'm not sure why these SG_ vars even need to be here since they're only used in the SG?

Reply:
Unsure, but that's the way it is across all frontends. If we change it, we should do it across all FEs. Let's do this separately